tasks/aiida-prepare.yml

```diff
@@ -75,9 +75,9 @@
   pip:
     name:
     - pip~=20.0
-    - setuptools~=36.2
+    - setuptools
     - wheel
-    - numpy==1.17.4
+    - numpy  # required by pymatgen
```

Comment:
no pinning, are you sure ;-)

I'd feel safer with pinning it to some recent minor version

Reply:
It is only needed for the pymatgen install, it will be updated, if necessary, by the pip install of aiida-core. If you pin it, then you have to ensure it is compatible with aiidalab and aiida-core (a current issue in the full quantum mobile build)